mempool/clist_mempool_test.go

```diff
@@ -426,6 +426,11 @@ func TestMempoolMaxMsgSize(t *testing.T) {
 	mempl, cleanup := newMempoolWithApp(cc)
 	defer cleanup()
 
+	var (
+		maxMsgSize = 1024 * 1024
```

Comment:
can't we use `mempl.config.MaxMsgBytes` here?

Reply:
Yes, we can. I fixed this at https://github.com/tendermint/tendermint/pull/3826/commits/7e9a3318b06be6e26cf12f820a8efb9baf0f7dff.